package.json

```diff
@@ -55,21 +57,39 @@
 		"signal-exit": "^3.0.2",
 		"slice-ansi": "^3.0.0",
 		"string-length": "^3.1.0",
+		"typescript": "3.8.3",
 		"widest-line": "^3.1.0",
 		"wrap-ansi": "^6.2.0",
-		"yoga-layout-prebuilt": "^1.9.3"
+		"yoga-layout-prebuilt": "1.9.5"
 	},
 	"devDependencies": {
+		"@ava/babel": "1.0.1",
+		"@ava/typescript": "1.1.1",
 		"@babel/cli": "^7.8.3",
 		"@babel/core": "^7.8.3",
-		"@babel/plugin-proposal-class-properties": "^7.8.3",
-		"@babel/plugin-proposal-object-rest-spread": "^7.8.3",
+		"@babel/plugin-proposal-class-properties": "7.8.3",
+		"@babel/preset-env": "7.8.7",
 		"@babel/preset-react": "^7.8.3",
+		"@babel/preset-typescript": "7.8.3",
+		"@types/is-ci": "2.0.0",
+		"@types/lodash.throttle": "4.1.6",
+		"@types/node": "13.9.1",
 		"@types/react": "^16.9.19",
-		"ava": "^2.4.0",
+		"@types/react-reconciler": "0.18.0",
+		"@types/scheduler": "0.16.1",
+		"@types/signal-exit": "3.0.0",
+		"@types/sinon": "7.5.2",
+		"@types/slice-ansi": "2.0.0",
+		"@types/wrap-ansi": "3.0.0",
+		"@types/yoga-layout": "1.9.1",
+		"@typescript-eslint/eslint-plugin": "2.24.0",
+		"@typescript-eslint/parser": "2.24.0",
+		"ava": "^3.5.0",
 		"babel-eslint": "^10.0.1",
 		"delay": "^4.1.0",
+		"eslint-config-xo": "0.29.1",
 		"eslint-config-xo-react": "^0.22.0",
+		"eslint-config-xo-typescript": "0.26.0",
```

Comment:
Upgrade to the latest version `xo`. It has builtin TS support. No config needed. 👌

Reply:
Done!